src/test/java/io/confluent/examples/streams/SumLambdaIntegrationTest.java

```diff
@@ -80,7 +70,7 @@ public void shouldSumEvenNumbers() throws Exception {
     // Use a temporary directory for storing state, which will be automatically removed after the test.
     streamsConfiguration.put(StreamsConfig.STATE_DIR_CONFIG, TestUtils.tempDirectory().getAbsolutePath());
 
-    final KStream<Integer, Integer> input = builder.stream(inputTopic);
+    final KStream<Void, Integer> input = builder.stream(inputTopic);
```

Comment:
Our builder isn't type-safe when you use default serdes.

Reply:
Would `Object` be a better type instead of `Void` ?

Reply:
Object would be more relaxed. The idea here is to document that the keys are always `null` at this point. (`null`) is the only possible "value" of a `Void` reference.

Reply:
But it's just documentation, since this type is never checked.

Reply:
Ack. I am fine either way. Was just a thought.